### PR TITLE
[merged] Create policy file if it does not exists.

### PR DESCRIPTION
**Pull request by rhatdan**
Currently the policy file does not exist the tools fail, and the user
is scrambling to setup a correct policy file.

### CODE REVIEW

**Comment by rhatdan** [comment]
@aweiteka PTAL

**Comment by rhatdan** [comment]
Fixes https://github.com/projectatomic/atomic/issues/625

File: Atomic/trust.py [hunk 1]
```diff
@@ -120,8 +123,13 @@ def delete(self):
...
 
     def default(self):
-        with open(self.policy_filename, 'r+') as policy_file:
-            policy = json.load(policy_file)
+        mode = "r+" if os.path.exists(self.policy_filename) else "w+"
+        with open(self.policy_filename, mode) as policy_file:
+            if mode == "r+":
+                policy = json.load(policy_file)
+            else:
+                policy = { "default":[{}] }
```
Comment (aweiteka) [inline Atomic/trust.py]: I think we need the default type. See https://github.com/projectatomic/skopeo/blob/master/default-policy.json

```
{ "default": [ { "type": "insecureAcceptAnything" }] }
```

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**